src/main/java/mate/service/AuthenticationServiceImpl.java

```diff
@@ -0,0 +1,26 @@
+package mate.service;
+
+import java.util.Optional;
+import mate.dao.DriverDao;
+import mate.lib.Inject;
+import mate.lib.Service;
+import mate.lib.exception.AuthenticationException;
+import mate.model.Driver;
+
+@Service
+public class AuthenticationServiceImpl implements AuthenticationService {
+    @Inject
+    private DriverDao driverDao;
+    
+    @Override
+    public Driver login(String login, String password) throws AuthenticationException {
+        Optional<Driver> driver = driverDao.findByLogin(login);
```

Comment:
Lets use DriverService for finding driver by login

Reply:
done